LncRNAanalysisPipe.nf

```diff
@@ -69,11 +58,11 @@ def print_white = {  str -> ANSI_WHITE + str + ANSI_RESET }
 
 //Help information
 // Nextflow  version
-version="v0.2.42"

```

Comment:
You can get this done using a manifest description nowadays. have a look at https://github.com/nf-core/rnaseq/blob/1cd5ab733a35727b01370de2ae8462d65f57fbb9/nextflow.config#L133 for reference!